src/models/connections.ts

```diff
@@ -94,6 +94,40 @@ const connections = (sequelize: Sequelize) => {
     return result.map((r: any) => r.toJSON() as ConnectionModel);
   };
 
+  const findAllFollowersByName = async (
+    name: string,
+    offset = 0,
+    limit = 20,
+    order: 'DESC' | 'ASC' = 'DESC'
+  ): Promise<ConnectionModel[]> => {
+    let result = await model.findAll({
+      attributes: ['creator'],
+      where: { name },
+      offset,
+      limit,
+      order: [['createdAt', order]],
+    });
+
+    return result.map((r: any) => r.toJSON().creator);
+  };
+
+  const findAllFollowingsByCreator = async (
+    creator: string,
+    offset = 0,
+    limit = 20,
+    order: 'DESC' | 'ASC' = 'DESC'
+  ): Promise<ConnectionModel[]> => {
+    let result = await model.findAll({
+      attributes: ['name'],
+      where: { creator },
+      offset,
+      limit,
+      order: [['createdAt', order]],
+    });
+
+    return result.map((r: any) => r.toJSON().name);
```

Comment:
name vs messageId?
see https://github.com/zkitter/zkitterd/pull/21#discussion_r1000470190

Reply:
nvm let's keep as is